Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -2576,6 +2576,26 @@ public McClellanSummationIndex MSI(Symbol[] symbols, int fastPeriod = 19, int sl
             return msi;
         }
 
+
+        /// <summary>
+        /// Creates a new RogersSatchellVolatility indicator for the symbol. The indicator will be automatically
+        /// updated on the given resolution.
+        /// </summary>
+        /// <param name="symbol">The symbol whose RogersSatchellVolatility we want</param>
+        /// <param name="period">The period of the rolling window used to compute volatility</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to casting the input value to a TradeBar</param>
+        /// <returns>A new RogersSatchellVolatility indicator with the specified smoothing type and period</returns>
+        [DocumentationAttribute(Indicators)]
+        public RogersSatchellVolatility RSVolat(Symbol symbol, int period, Resolution? resolution = null, Func<IBaseData, TradeBar> selector = null)
```

Comment:
Helper method name should be `RSV` I think? Same for string indicator names here an in the indicator `$"RSVolat({period})"`

Reply:
Done but using `$"RSV({period})"`